keras/src/wrappers/fixes.py

```diff
@@ -0,0 +1,119 @@
+import sklearn
+from packaging.version import parse as parse_version
+from sklearn import get_config
+
+sklearn_version = parse_version(parse_version(sklearn.__version__).base_version)
+
+if sklearn_version < parse_version("1.6"):
```

Comment:
If seems like it would be much easier and maintainable to simply require a minimum sklearn version?

Reply:
it's not the worst. And I've included all version specific code in a single `fixes.py` so that we know later where to clean up. Also, 1.6 was just released a few days ago, so I don't think it's a good idea to have that as a minimum required version. WDYT?